src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/talismans/MagicianTalisman.java

```diff
@@ -70,7 +71,7 @@ public TalismanEnchantment getRandomEnchantment(@Nonnull ItemStack item, @Nonnul
 
         // @formatter:off
         List<TalismanEnchantment> enabled = enchantments.stream()
-                .filter(e -> e.getEnchantment().canEnchantItem(item))
+                .filter(e -> item.getType() == Material.BOOK || e.getEnchantment().canEnchantItem(item))
```

Comment:
About the balancing: I'd perhaps suggest an `ItemSetting<Boolean>` to disable/enable whether books can be enchanted too.